scripts/lease.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+lease() {
+  eval $(k8s-claimer --server=k8s-claimer-e2e.deis.com lease create --duration=$CLUSTER_DURATION)
+  echo "Leased cluster $CLUSTER_NAME for $CLUSTER_DURATION seconds"
+  echo "TOKEN: $TOKEN"
+  if [ -z $TOKEN ]; then
+    echo "Lease failed exiting."
+    exit
+  fi
+  # We should exit if we cant get a cluster
```

Comment:
I'm pretty sure `exit` returns the last command's status code when no arguments are supplied, according to [this](http://tldp.org/LDP/abs/html/exit-status.html). Good catch.